src/vcs/git.rs

```diff
@@ -788,6 +789,55 @@ impl<'a> Browser<'a> {
         self.repository.revision_branches(&commit.id())
     }
 
+    /// Get the [`Stats`] of the underlying [`Repository`].
+    ///
+    /// # Errors
+    ///
+    /// * [`error::Error::Git`]
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use radicle_surf::vcs::git::{Browser, Repository};
+    /// # use std::error::Error;
+    ///
+    /// # fn main() -> Result<(), Box<dyn Error>> {
+    /// let repo = Repository::new("./data/git-platinum")?;
+    /// let mut browser = Browser::new(&repo)?;
+    ///
+    /// let stats = browser.get_stats()?;
+    ///
+    /// // there should be 5 branches (local & remote)
+    /// assert_eq!(stats.branch_count, 5);
+    ///
+    /// // there should be 14 commits
+    /// assert_eq!(stats.commit_count, 14);
+    ///
+    /// // 4 contributors: Alexander Simmerl, Fintan Halpenny, FintanH, Rudolfs Ošiņš
+    /// assert_eq!(stats.contributor_count, 4);
+    ///
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn get_stats(&self) -> Result<Stats, Error> {
+        let history = self.repository.head()?;
+
+        let branch_count = self.list_branches(None)?.len();
+        let commit_count = history.iter().count();
+        let mut contributors: Vec<String> =
+            history.iter().cloned().map(|x| x.author.name).collect();
```

Comment:
This is more a tip for type inference. You can also do `collect::<Vec<_>>()` the `_` will be inferred and you're telling the compiler that you're expecting a `Vec`.